server/migrations/20170328005425-create-game.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+    return queryInterface.createTable('Games', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      game_integer: {
+        type: Sequelize.INTEGER
+      },
+      sr_change: {
+        type: Sequelize.INTEGER
+      },
+      rank: {
+        type: Sequelize.INTEGER
+      },
+      streak: {
+        type: Sequelize.INTEGER
+      },
+      map: {
+        type: Sequelize.TEXT
+      },
+      score: {
+        type: Sequelize.STRING
+      },
+      group_size: {
+        type: Sequelize.INTEGER
+      },
+      heroes: {
+        type: Sequelize.TEXT
```

Comment:
For now, let's remove the heroes field, and just implement the MVP stuff

We can properly flesh out how to handle heros and the other stats later